ui/components/ui/contract-token-values/index.scss

```
@@ -0,0 +1,17 @@
+.contract-token-values {
+  &__copy-address {
+    margin-inline-start: 8px;
+
+    &__button {
+      padding: 0;
+    }
+  }
+
+  &__block-explorer {
+    margin-inline-start: 16px;
```

Sorry I should have suggested 8px here as well
```suggestion
    margin-inline-start: 8px;
```

---

Hi @georgewrmarshall Done [here](https://github.com/MetaMask/metamask-extension/pull/15617/commits/cfc287279869d91b7e1592d2166ecb5885fc9a17).